source/features/more-conversation-filters.tsx

```diff
@@ -32,7 +33,16 @@ function init(): void {
 	commentsLink.after(subscriptionsLink);
 }
 
+// The page below doesn't have the filters menu
+// https://github.com/tc39/proposal-await.ops/issues?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen
+async function hasFilters(): Promise<boolean> {
+	return Boolean(await elementReady('#filters-select-menu', {waitForChildren: false}));
```

Comment:
The feature has `awaitDromReady`, so it doesn't need `elementReady`

But here the fix is to just add `if (!sourceItem) return` on line 10, not this